libs/core/src/lib/list/list.component.scss

```diff
@@ -18,3 +18,29 @@
         border-bottom: var(--sapList_BorderWidth, 0.0625rem) solid var(--sapList_SelectionBorderColor, #0854a0);
     }
 }
+
+.fd-list .fd-list__group-header {
+    &:focus,
+    &.is-focus {
+        outline: none;
+        position: relative;
+
+        &::before {
+            content: '';
+            position: absolute;
+            display: block;
+            border: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
+            top: 0.125rem;
+            left: 0.125rem;
+            right: 0.125rem;
+            bottom: 0.125rem;
+        }
+
+        &:active,
+        &.is-active {
+            &::before {
+                border: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_ContrastFocusColor);
+            }
+        }
+    }
+}
```

Comment:
What if you just add `fd-list__item` class to `ListGroupHeaderDirective` element, instead of having css code like that?